File: java/jetty/src/main/java/ExtremeStartupServer.java
import org.eclipse.jetty.server.Server;
import org.eclipse.jetty.webapp.WebAppContext;

public class ExtremeStartupServer {

    public static void main(String[] args) throws Exception {
        Server server = new Server(1337);
        server.setHandler(new WebAppContext("src/main/webapp", "/"));
        server.start();
    }

}

File: scala/scalatra/src/main/scala/ExtremeStartupServer.scala
import org.eclipse.jetty.server.Server
import org.eclipse.jetty.servlet.ServletContextHandler
import org.eclipse.jetty.servlet.ServletHolder
    
object ExtremeStartupServer {

    def main(args:Array[String]) = {
        val server = new Server(1337)
        val root = new ServletContextHandler(server, "/", ServletContextHandler.SESSIONS)
        root.addServlet(new ServletHolder(new ExtremeStartup), "/*")
        server.start()
    }

}
